prow/getting_started_kind_minikube.md

```diff
@@ -0,0 +1,109 @@
+# Simplified kind/minikube deployment 
+
+This document is targeting developers, testers, and all the other interested parties who would like to learn or test prow, but do not have resources other than a localhost workstation. It describes a simplified minikube/kind deployment of crucial prow components:
+
+* [`crier`](prow/cmd/crier)
```

Comment:
Aren't these links relative to the current directory (`prow`)? Shouldn't they be `cmd/deck` etc instead? At least the links do not work for me in https://github.com/jmguzik/test-infra/blob/kind-minikube/prow/getting_started_kind_minikube.md